feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/vif.c

```diff
@@ -211,119 +215,154 @@ static void vif_state_security_get(struct schema_Wifi_VIF_State *vstate,
-	uci_to_blob(&b, s, &wifi_iface_param);
-	blobmsg_parse(wifi_iface_policy, __WIF_ATTR_MAX, tb, blob_data(b.head), blob_len(b.head));
-
-	if (!tb[WIF_ATTR_DEVICE] || !tb[WIF_ATTR_IFNAME]) {
-		LOGE("%s: invalid radio/ifname", s->e.name);
-		return false;
-	}
-	ifname = target_unmap_ifname(blobmsg_get_string(tb[WIF_ATTR_IFNAME]));
-	strncpy(radio, blobmsg_get_string(tb[WIF_ATTR_DEVICE]), IF_NAMESIZE);
-
-	vstate._partial_update = true;
-	vstate.associated_clients_present = false;
-	vstate.vif_config_present = false;
-
-	SCHEMA_SET_INT(vstate.rrm, 1);
-	SCHEMA_SET_INT(vstate.btm, 1);
-	SCHEMA_SET_INT(vstate.vlan_id, 1);
-	SCHEMA_SET_INT(vstate.ft_psk, 0);
-	SCHEMA_SET_INT(vstate.group_rekey, 0);
-
-	strscpy(vstate.mac_list_type, "none", sizeof(vstate.mac_list_type));
-	vstate.mac_list_len = 0;
-	vstate.mac_list_type_exists = true;
-
-	SCHEMA_SET_STR(vstate.if_name, ifname);
-
-	if (tb[WIF_ATTR_HIDDEN] && blobmsg_get_bool(tb[WIF_ATTR_HIDDEN]))
-		SCHEMA_SET_STR(vstate.ssid_broadcast, "disabled");
-	else
-		SCHEMA_SET_STR(vstate.ssid_broadcast, "enabled");
-
-	if (tb[WIF_ATTR_MODE])
-		SCHEMA_SET_STR(vstate.mode, blobmsg_get_string(tb[WIF_ATTR_MODE]));
-	else
-		SCHEMA_SET_STR(vstate.mode, "ap");
-
-	if (tb[WIF_ATTR_DISABLED] && blobmsg_get_bool(tb[WIF_ATTR_DISABLED]))
-		SCHEMA_SET_INT(vstate.enabled, 0);
-	else
-		SCHEMA_SET_INT(vstate.enabled, 1);
-
-	if (tb[WIF_ATTR_ISOLATE] && blobmsg_get_bool(tb[WIF_ATTR_ISOLATE]))
-		SCHEMA_SET_INT(vstate.ap_bridge, 1);
-	else
-		SCHEMA_SET_INT(vstate.ap_bridge, 0);
-
-//	if (tb[WIF_ATTR_UAPSD] && blobmsg_get_bool(tb[WIF_ATTR_UAPSD]))
-		SCHEMA_SET_INT(vstate.uapsd_enable, true);
-//	else
-//		SCHEMA_SET_INT(vstate.uapsd_enable, false);
-
-	if (tb[WIF_ATTR_NETWORK])
-		SCHEMA_SET_STR(vstate.bridge, blobmsg_get_string(tb[WIF_ATTR_NETWORK]));
-	else
-		LOGW("%s: unknown bridge/network", s->e.name);
-
-	if (vstate.bridge_exists &&
-	    (p = strstr(vstate.bridge, "vlan")) != NULL) {
-		long int v = strtol(&p[4], NULL, 10);
-
-		SCHEMA_SET_INT(vstate.vlan_id, v);
-	} else
-		LOGW("%s: Cannot get VlanId", s->e.name);
-
-	if (tb[WIF_ATTR_SSID])
-		SCHEMA_SET_STR(vstate.ssid, blobmsg_get_string(tb[WIF_ATTR_SSID]));
-	else
-		LOGW("%s: failed to get SSID", s->e.name);
-
-	if (strstr(s->e.name, "50"))
-		SCHEMA_SET_STR(vstate.min_hw_mode, "11ac");
-	else
-		SCHEMA_SET_STR(vstate.min_hw_mode, "11n");
-
-	if (tb[WIF_ATTR_BSSID])
-		SCHEMA_SET_STR(vstate.mac, blobmsg_get_string(tb[WIF_ATTR_BSSID]));
-	else if (tb[WIF_ATTR_IFNAME] && !vif_get_mac(blobmsg_get_string(tb[WIF_ATTR_IFNAME]), mac))
-		SCHEMA_SET_STR(vstate.mac, mac);
-	else
-		LOGN("%s: Failed to get base BSSID (mac)", vstate.if_name);
-
-	vif_state_security_get(&vstate, tb);
-
-	if (vconf) {
-		LOGN("%s: updating vif config", radio);
-		vif_state_to_conf(&vstate, vconf);
-		radio_ops->op_vconf(vconf, radio);
-	}
-	LOGN("%s: updating vif state", radio);
-	radio_ops->op_vstate(&vstate, radio);
-
-	return true;
+      struct blob_attr *tb[__WIF_ATTR_MAX] = { };
```

Comment:
Hi,
your editor has reformatted a whole function again. What editor are you using if I may ask ?
   John

Reply:
I’m using Eclipse ,it has default indentation styles. It is changing indentation styles based on that. I’ll figure it out and make changes to the default settings

Reply:
Hi John,
I’m using Eclipse ,it has default indentation styles. It is changing indentation styles based on that. I’ll figure it out and make changes to default settings.
Nagendrababu.

Reply:
FYI, according to [Opensync Coding Standard](https://static1.squarespace.com/static/5bbce542d7819e023f203f03/t/5e9755e7e6c4930b7081c265/1586976261532/EDE-020-041-501_OpenSync_Coding_Standard.pdf):

> Do not use tabs
> Never use tabs in the code.
> While editors can be configured to have tab stops at 4 instead of 8 columns, other tools (like
> visual diff tools) might not all be configurable.

Reply:
This is for editing the core code. we are building our own platform code however. the current platform code i wrote follows the kernel coding style and is c99 compliant. This is what opensource folks would expect to find and aligns with the entire openwrt code base.

Reply:
independent of that the problem here is that the ecplise editor reformats whole functions making diff impossible to review

Reply:
> This is for editing the core code. we are building our own platform code however. the current platform code i wrote follows the kernel coding style and is c99 compliant. This is what opensource folks would expect to find and aligns with the entire openwrt code base.

Is [this](https://www.kernel.org/doc/html/v4.10/process/coding-style.html) the coding style guide to follow for code in platform/openwrt?

Reply:
That would be ideal. In general I am happy to quickly clean stuff up when merging it into trunk. This is not a problem at all and happy to do so. The issue with this specific patch is that it is not obvious what was actually changed in the function as the whole function gets removed and replaced with a different one. Hope that makes sense and is ok for everyone.

Reply:
I tried to change the indentation style to align with code base after my editor changed to the default indentation style, but I've ended up like I've changed whole function.

Reply:
> That would be ideal. In general I am happy to quickly clean stuff up when merging it into trunk. This is not a problem at all and happy to do so. The issue with this specific patch is that it is not obvious what was actually changed in the function as the whole function gets removed and replaced with a different one. Hope that makes sense and is ok for everyone.

can you hide whitespace changes in the settings while reviewing?

Reply:
https://github.com/PanderMusubi/linux-kernel-code-style-formatters  this likes an ecplise plugin for the style formatter